test/MultipleWinners.test.js

```diff
@@ -85,108 +119,454 @@ describe('MultipleWinners', function() {
+          token: 0,
+        },
+        {
+          target: wallet6.address,
+          percentage: 300,
+          token: 0,
+        },
+        {
+          target: wallet5.address,
+          percentage: 150,
+          token: 0,
+        },
+      ])
+
+      const prizeSplits = await prizeStrategy.prizeSplits();
+      expect(prizeSplits[0].percentage)
+      .to.equal(50)
+      expect(prizeSplits[1].percentage)
+      .to.equal(300)
+      expect(prizeSplits[2].percentage)
+      .to.equal(150)
+    });
+
+    it("should set two split prize configs, update the first and remove the second prize split config", async () => {
+      await prizeStrategy.setPrizeSplits([
+        {
+          target: wallet5.address,
+          percentage: 50,
+          token: 0,
+        },
+        {
+          target: wallet6.address,
+          percentage: 500,
+          token: 0,
+        },
+      ]);
+
+      await prizeStrategy.setPrizeSplits([
+        {
+          target: wallet5.address,
+          percentage: 400,
+          token: 0,
+        },
+      ]);
+
+      const prizeSplits = await prizeStrategy.prizeSplits();
+      expect(prizeSplits.length)
+      .to.equal(1)
+      expect(prizeSplits[0].percentage)
+      .to.equal(400)
+    });
+
+    it("should set two split prize configs and a remove all prize split configs", async () => {
+      await prizeStrategy.setPrizeSplits([
+        {
+          target: wallet5.address,
+          percentage: 50,
+          token: 0,
+        },
+        {
+          target: wallet6.address,
+          percentage: 500,
+          token: 0,
+        },
+      ]);
+      await expect(prizeStrategy.setPrizeSplits([]))
+      .to.emit(prizeStrategy, "PrizeSplitRemoved").withArgs(0)
+
+      const prizeSplits = await prizeStrategy.prizeSplits();
+      expect(prizeSplits.length)
+      .to.equal(0)
+    });
+  });
+
+  describe("distribute()", () => {
+    it("should ignore awarding prizes if there are no winners to select", async () => {
+      await prizePool.mock.captureAwardBalance.returns(toWei("10"));
+      await ticket.mock.draw.withArgs(10).returns(ethers.constants.AddressZero);
+      await expect(prizeStrategy.distribute(10)).to.emit(
+        prizeStrategy,
+        "NoWinners"
+      );
+    });
+
+    it("should award a single winner", async () => {
+      await prizeStrategy.setNumberOfWinners(1);
+
+      let randomNumber = 10;
+      await prizePool.mock.captureAwardBalance.returns(toWei("8"));
+      await ticket.mock.draw.withArgs(randomNumber).returns(wallet3.address);
+
+      await externalERC20Award.mock.balanceOf
+        .withArgs(prizePool.address)
+        .returns(0);
+
+      await ticket.mock.totalSupply.returns(1000);
+
+      await prizePool.mock.award
+        .withArgs(wallet3.address, toWei("8"), ticket.address)
+        .returns();
+
+      await prizeStrategy.distribute(randomNumber);
+    });
+
+    it("should award prize splits to multiple targets", async () => {
```

Comment:
Once this is moved to a PrizeSplits.test.js and _distributePrizeSplits is exposed via a harness it'll be much easier to test every variation